louiseA/file-1.js

```diff
@@ -0,0 +1,103 @@
+1.
+
+ a) "1" == 1
+  true
+
+ b) "1" === 1
+  false
+
+ c) x == "x"
+  true
+
+ d) x == (x+'')
+  false
+
+ e) '' == ' '
+  true
+
+ f) x = true
+  true
+
+ g) var x; x == 'undefined'
+  false
+
+ h) '9' > '10'
+  true
+
+ i) typeof x + 1 === 'number'
+  false
+
+ j) typeof x % 2 === 'number'
+  false
+
+ k) typeof (x % 2) === 'number'
+  true
+
+ l) x++ == ++x //having trouble understanding the increment operator...
+
```

Comment:
Here's the basic idea:

Consider that x = 1.

x++ returns the value of x before adding 1 to x. So on the left side you end up with 1 == something.

++x adds 1 to the value of x (which is now 2, because of the left hands x++) before returning the value. So what we end up on the right side is 3.

This brings us to evaluate 1 == 3, which is not true.